### PR TITLE
First branch pull request

### PR DESCRIPTION
**Pull request by Gayathrik05**
Hi Pravin,
        I'm Gayathri. I have front end & back end files in this micro project repository, and also i have created new branch and created some required files for my micro project. So Please verify my files. If you find any mistakes, kindly give your solutions. 
Thank you...

### CODE REVIEW

File: Employee.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+	public String getEmpBg() {
+		return empBg;
+	}
+
+	public void setEmpBg(String empBg) {
+		this.empBg = empBg;
+	}
+
+
```
Comment (PravinKumarRajaSahayam) [inline Employee.java:91]: Add a getter and setter for joining date and job title field

**Review by PravinKumarRajaSahayam — COMMENTED** [review]
Add a getter and setter for joining date and job title field

**Review by Gayathrik05 — COMMENTED** [review]
I updated my Employee.java file. please recheck it.

File: Employee.java (hunk 1 above)
Comment (Gayathrik05) [reply Employee.java:91]: I updated my Employee.java file. please recheck it.

File: Message.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+	public String getEmployeeMessage() {
+		return employeeMessage;
+	}
+
+	public void setEmployeeMessage(String employeeMessage) {
+		this.employeeMessage = employeeMessage;
+	}
+
+
```
Comment (PravinKumarRajaSahayam) [inline Message.java:63]: Add a toString to your Message.java file

File: Message.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	@Column(name = "employeeEmail")
+	private String employeeEmail;
+
+	@Column(name = "employeeMessage")
+	private String employeeMessage;
+
+	public Message() {
+		super();
+	}
+
```
Comment (PravinKumarRajaSahayam) [inline Message.java:28]: Add a parameterized constructor in your Message.java

**Review by PravinKumarRajaSahayam — COMMENTED** [review]
Add a code to your Message.java

File: Message.java (hunk 3 above)
Comment (Gayathrik05) [reply Message.java:28]: Okay. I will update my file. Please recheck it.

File: Message.java (hunk 2 above)
Comment (Gayathrik05) [reply Message.java:63]: Okay. I will update method in my file.

**Review by Gayathrik05 — COMMENTED** [review]
I will update my missing details in my file. Please recheck it

File: Message.java (hunk 3 above)
Comment (PravinKumarRajaSahayam) [reply Message.java:28]: Ok

**Review by PravinKumarRajaSahayam — COMMENTED** [review]
Ok

File: Users.java [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+	@Column(name = "User_Password")
+	private String userPassword;
+	
+	@Column(name = "User_Confirm_Password")
+	private String userConfirmPassword;
+	
+	@Column(name="User_Type")
+	private String userType;
+	
+	@OneToOne(cascade = CascadeType.ALL)
```
Comment (PravinKumarRajaSahayam) [inline Users.java:37]: Remove this three lines

File: Users.java [hunk 5]
```diff
@@ -0,0 +1,124 @@
+package com.abp.bean;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
```
Comment (PravinKumarRajaSahayam) [inline Users.java:9]: Remove this 3, 9, 10 lines

**Review by PravinKumarRajaSahayam — COMMENTED** [review]
Remove this lines

File: Users.java (hunk 4 above)
Comment (Gayathrik05) [reply Users.java:37]: Okay. I remove these lines.

File: Users.java (hunk 5 above)
Comment (Gayathrik05) [reply Users.java:9]: Okay. I remove unnecessary import statements.

**Review by Gayathrik05 — COMMENTED** [review]
Please recheck it

File: Users.java (hunk 4 above)
Comment (Gayathrik05) [reply Users.java:37]: Okay. I removed these lines in my users.java file

File: Users.java (hunk 5 above)
Comment (Gayathrik05) [reply Users.java:9]: I removed unnecessary import statements.

**Comment by PravinKumarRajaSahayam** [comment]
I approved your content

**Comment by Gayathrik05** [comment]
Please review my updated files and give permission to merge a branch into main.

**Comment by Gayathrik05** [comment]
Please review my updated files and give permission to merge a branch into main.

**Pull request merged**